ITI.SFML.Graphics/IRenderTarget.cs

```diff
@@ -17,17 +17,9 @@ public interface IRenderTarget
         /// </summary>
         View DefaultView { get; }
 
-        /// <summary>
-        /// Return the current active view
-        /// </summary>
-        /// <returns>The current view</returns>
-        View GetView();
 
-        /// <summary>
-        /// Changes the current active view.
-        /// </summary>
-        /// <param name="view">New view</param>
-        void SetView( View view );
+
+        View View { get; set; }
```

Comment:
What's the difference between DefaultView and View/(Get/SetView)?
Isn't there a reason for this 2 different API point?

Reply:
SFML docs say: 
Get the default view of the render target.

The default view has the initial size of the render target, and never changes after the target has been created.